app/assets/stylesheets/_forms.scss

```diff
@@ -8,7 +8,7 @@ fieldset {
 input,
 label,
 select {
-  display: block;
+  display: inline;
```

Comment:
I didn't see where all of those text inputs would get the block below. I guess my only concern here would be why we would want inline labels and selects then.